server/src/main/java/io/crate/execution/engine/window/WindowProjector.java

```diff
@@ -155,6 +157,8 @@ public static Projector fromProjection(WindowAggProjection projection,
         return sourceRows -> WindowFunctionBatchIterator.of(
             sourceRows,
             ramAccounting::addBytes,
+            // Same as RamUsageEstimator.shallowSizeOf(array) but without NUM_BYTES_ARRAY_HEADER as we are accounting only for expansion.
+            (deltaLength) -> ramAccounting.addBytes(RamUsageEstimator.alignObjectSize((long) NUM_BYTES_OBJECT_REF * deltaLength)),
```

Comment:
The reason I'm not using `addBytes` (which we already pass to window function, added in https://github.com/crate/crate/pull/16054/commits/0b29ba969a6ccd26d5ad7327d511e0ec42a3d4ce) is that `AccountableList` is in shared libs and don't see server dependencies (RamUsageEstimator). 

Other options
1.  move AccountableList to server
2. inline NUM_BYTES_OBJECT_REF constant into AccoutableList and also inline 
   RamUsageEstimator.alignObjectSize
3. Keep as is
 
 I would prefer 2 as passing similar functions doesn't look nice. One caveat  - once we start introducing more data structure, we can end up inlining more stuff from `RamUsageEstimator`.

Reply:
Another argument for 2 is https://github.com/crate/crate/pull/16263#discussion_r1665351199.
Ie we sometimes need more complex accounting, it's better to have generic `addBytes` as accounting function and do any potentially complex logic in the data structure.

Reply:
As discussed, will move to server (go with 1). 

Similar to 2, it allows to re-use `addBytes` but has an advantage that we don't have to inline/copy paste a  lot of code.